tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/four_photo_collage_block.html

```diff
@@ -10,10 +10,14 @@
             {% endfor %}
         </div>
     </div>
-    {% if value.caption %}
-        <div class="heading heading--three heading--light four-photo-collage__caption rich-text">{{ value.caption|richtext }}</div>
-    {% endif %}
-    {% if value.description %}
-        {{ value.description|richtext }}
-    {% endif %}
+    <div class="four-photo-collage__text">
```

Comment:
Suggestion: Could wrap this in `{% if value.caption or value.description %}` so we don't output an empty `<div>` if they're both false. Not sure how likely that would be?

Reply:
thank you, @chris-lawton, it's my oversight - it certainly could be the case
committed now!